src/js/view/components/LogViewerList.js

```diff
@@ -92,13 +92,8 @@ const LogViewerList = props => {
     let currentMaxLength = maxLineLength;
     let index = lastLineCount;
     for (; index < props.lines.length; index++) {
-      // Remove all of the stuff hiddenWindow has added to it, as they shouldn't count towards the length of the string
-      let lineWithoutExtrasLength = props.lines[index].replace(
-        /\[\/?HL[LG\d]+\]/g,
-        ''
-      ).length;
-      if (lineWithoutExtrasLength > currentMaxLength) {
-        currentMaxLength = lineWithoutExtrasLength;
+      if (props.lines[index].length > currentMaxLength) {
```

Comment:
Everything seems to work really well on my machine and I like that many parts of the code, like this, got a lot cleaner. 
I think that @Atmanea is working on a bug that might concern some of this code in the LogViewerList, if I am not remembering it wrong. So I'd like to have her, and maybe someone else check it out and try it on their machine before I approve anything:)

Reply:
Yes, the length calculations of a line will not be necessary to calculate. At the moment it is only used to set the width for a single log line but this can be done dynamically so it is not necessary. However, my branch is not ready to be implemented just yet so go for this for now and I'll change it when it's time :)